WarSpecialists/Assets/Scripts/PlayerMovement.cs

```diff
@@ -4,9 +4,21 @@
 
 public class PlayerMovement : MonoBehaviour
```

Comment:
Why did you change PlayerMovement? Do you have the most updated branch? You have conflicts as well in this class. Fix them or don't push this changes.